requirements_test_pre_commit.txt

```diff
@@ -1,9 +1,5 @@
 -r requirements_test_min.txt
 black==23.1.0
```

Comment:
I'd be okay with removing this as well

Reply:
Removing black or removing the version ?

Reply:
versioN!